symbolic-debuginfo/Cargo.toml

```diff
@@ -15,43 +15,98 @@ as Mach-O or ELF.
 """
 edition = "2018"
 
-exclude = [
-    "tests/**/*",
-]
+exclude = ["tests/**/*"]
 
 [package.metadata.docs.rs]
 all-features = true
 
+[features]
+default = ["breakpad", "elf", "macho", "ms", "sourcebundle", "wasm"]
```

Comment:
Yah, this was definitely a bit of a weird split, but I think it makes sense because pdb and pe would presumably always go together, whereas dwarf can be used separately with both elf, macho, and wasm. That being said I could of course split it into two features.